web/src/components/event-cards-query.js

```diff
@@ -0,0 +1,72 @@
+import EventCards from './event-cards';
+import {gql} from 'apollo-boost';
+import {Query} from "react-apollo";
+import React, {Component} from 'react';
+
+class EventCardsQuery extends Component {
+  constructor(props) {
+    /**
+     * Props
+     * pageSize {int} : Number of event card that bring from server once
+     * selectedTagIds {Array<Int>} : Array of selected tag id
+     * isRecommended {boolean} : whether recommend or not (current not use)
+     * after {int} :
+     * onCreate : HandlerCurrentCursor function of parent(eventlist.js)
+     * */
```

Comment:
Could you write JSDoc like this?
/**
  * @param {int} pageSize: decription
...
...
* */